### PR TITLE
feat(web): rename project type values

### PR DESCRIPTION
**Pull request by m8vago**
Rename project type values:
- simple -> versionless
- complex -> versioned

### CODE REVIEW

File: web/crux-ui/e2e/image-config.spec.ts [hunk 1]
```diff
@@ -7,11 +7,11 @@ import { waitSocket, waitSocketReceived, waitSocketSent } from './utils/websocke
 
 const setup = async (
   page: Page,
-  projectNAme: string,
```
Comment (robot9706) [inline web/crux-ui/e2e/image-config.spec.ts:10]: 🦅

File: web/crux-ui/e2e/nodes.spec.ts [hunk 2]
```diff
@@ -1,231 +1,231 @@
 import { deploymentContainerLogUrl, deploymentDeployUrl, nodeContainerLogUrl, ROUTE_NODES } from '@app/routes'
 import { expect, test } from '@playwright/test'
 import { DAGENT_NODE, screenshotPath } from './utils/common'
-import { addDeploymentToSimpleProject, addImageToSimpleProject, createProject } from './utils/projects'
+import { addDeploymentToVersionlessProject, addImageToVersionlessProject, createProject } from './utils/projects'
 
-test('Install dagent should be successful', async ({ page }) => {
-  await page.goto(ROUTE_NODES)
+// test('Install dagent should be successful', async ({ page }) => {
```
Comment (robot9706) [inline web/crux-ui/e2e/nodes.spec.ts:6]: Why is this commented?

File: web/crux-ui/e2e/version.spec.ts [hunk 3]
```diff
@@ -72,12 +72,12 @@ test('Change default version should work', async ({ page }) => {
...
   await expect(imagesRows).toHaveCount(1)
-  await expect(await page.locator('div.table-cell:has-text("redis")').first()).toBeVisible()
-  await expect(await page.locator('div.table-cell:has-text("nginx")')).toHaveCount(0)
+  await expect(await page.locator('.table-cell:has-text("redis")').first()).toBeVisible()
+  await expect(await page.locator('.table-cell:has-text("nginx")')).toHaveCount(0)
 })
 
 test('Increase version should work', async ({ page }) => {
-  const projectId = await createProject(page, 'PW-INCREASE-VERSION', 'Complex')
+  const projectId = await createProject(page, 'Pincrease-version', 'versioned')
```
Comment (robot9706) [inline web/crux-ui/e2e/version.spec.ts:80]: `Pincrease`? 🤗

**Review by polaroi8d — APPROVED** [review]
Minor comments, tested locally works fine.

File: web/crux-ui/e2e/nodes.spec.ts (hunk 2 above)
Comment (m8vago) [reply web/crux-ui/e2e/nodes.spec.ts:6]: Whoopsy.

**Pull request merged**